### PR TITLE
Fixes #28268 - change v4 to be the default in ovirt cr

### PR DESCRIPTION
**Pull request by yifatmakias**

### CODE REVIEW

**Review by shiramax — DISMISSED** [review]

File: app/models/compute_resources/foreman/model/ovirt.rb
Comment (ekohl) [inline app/models/compute_resources/foreman/model/ovirt.rb]: This effectively makes it impossible to use v3, right? It's effectively the same as:
```ruby
def use_v4
    true
end
```

I think this should work:
```suggestion
      self.attrs.fetch(:ovirt_use_v4, true)
```

**Comment by yifatmakias** [comment]
[test foreman]

File: app/models/compute_resources/foreman/model/ovirt.rb
Comment (yifatmakias) [reply app/models/compute_resources/foreman/model/ovirt.rb]: Got It, Thanks! :)

**Comment by yifatmakias** [comment]
[test foreman]

**Comment by yifatmakias** [comment]
[test foreman]

**Comment by shiramax** [comment]
[ test foreman ]

**Comment by yifatmakias** [comment]
[test foreman]

**Comment by yifatmakias** [comment]
This PR depends on a fog oVirt PRs:
https://github.com/fog/fog-ovirt/pull/84
https://github.com/fog/fog-ovirt/pull/87
Foreman test will not pass until this PR will be merged and released in the next fog-oVirt release.

**Comment by yifatmakias** [comment]
[test foreman]

**Comment by yifatmakias** [comment]
[test foreman]

**Comment by yifatmakias** [comment]
[test foreman]

**Comment by shiramax** [comment]
I released a new fog-ovirt version, @yifatmakias  please bump ovirt version to 1.2.4 
packaging-pr: https://github.com/theforeman/foreman-packaging/pull/4512

**Comment by yifatmakias** [comment]
[test foreman]

**Comment by yifatmakias** [comment]
[test katello]

**Review by shiramax — APPROVED** [review]

**Review by shiramax — APPROVED** [review]

**Comment by shiramax** [comment]
@ekohl can you please approve this PR? I can't merge it because it's waiting for  theforeman/packaging owner review.

**Review by ekohl — APPROVED** [review]
1.2.4 is in nightly so :+1: from the packaging side.

**Comment by shiramax** [comment]
thanks @ekohl !

**Pull request merged**
